### ISSUE TITLE
add spawn out button

### ISSUE THREAD

**Issue by wsor4035** [BlockCityCreative/bc_rules#1]
add a button to the /rules formspec to teleport the player to spawn out

**Comment by samarth-ramesh** [comment]
maybe not on the /rules formspec, but on the formspec shown to new players?

it could create something that allows playersr to tp at will to spawn out

**Comment by wsor4035** [comment]
> maybe not on the /rules formspec, but on the formspec shown to new players?
/rules formspec == formspec shown to new players

**Comment by samarth-ramesh** [comment]
the /rules formspec is what is shown when a player runs /rules... but the newplayer formspec can be shown only once to each player

**Comment by wsor4035** [comment]
closing this as this mod has been merged into bc_custome
